src/services/lease-service/index.ts

```diff
@@ -1149,12 +1149,24 @@ export const routes = (router: KoaRouter) => {
    */
   router.get('/listing/:listingId/applicants/details', async (ctx) => {
     const metadata = generateRouteMetadata(ctx)
-    const responseData =
-      await leasingAdapter.getListingByIdWithDetailedApplicants(
-        ctx.params.listingId
-      )
+    const result = await leasingAdapter.getDetailedApplicantsByListingId(
+      Number(ctx.params.listingId)
+    )
 
-    ctx.body = { content: responseData, ...metadata }
+    if (!result.ok) {
+      if (result.err === 'not-found') {
```

Comment:
We had a discussion about passing down the status code from the adapter last week. Could we apply this here and just proxy everything?

Reply:
> What about add those error codes to this endpoint when we need them? I did actually re-write the endpoint but adding error codes here and in leasing feels a bit out of scope.


If you think this is valid then perhaps this is out of scope for this particular endpoint here too?

Reply:
Did however use error codes on the adapter method i introduced
https://github.com/Bostads-AB-Mimer/onecore-core/pull/190/commits/83ee4c4d42e70d967b37170c2793a2189cc9ab61